src/Integration.Vsix/SonarLintDaemonPackage.cs

```diff
@@ -93,30 +91,7 @@ private async System.Threading.Tasks.Task InitAsync()
 
                 cFamilyPreCompiledHeadersEventListener = await this.GetMefServiceAsync<IPreCompiledHeadersEventListener>();
 
-                daemon = await this.GetMefServiceAsync<ISonarLintDaemon>();
```

Comment:
This package still has non-daemon related bits in it. So far I've deleted anything that references the ISonarLintDaemon or installer directly and left everything else.